iroh-relay/src/server/client_conn.rs

```diff
@@ -354,10 +358,10 @@ enum State {
 }
 
 impl RateLimitedRelayedStream {
-    fn new(inner: RelayedStream, limiter: governor::DefaultDirectRateLimiter) -> Self {
+    fn new(inner: RelayedStream, limiter: Option<governor::DefaultDirectRateLimiter>) -> Self {
```

Comment:
maybe two apis, `RateLimitedRelayedStream::unlimited(RelayedStream)` and `RateLimitedRelayedStream::new(RelayedStream, RateLimiter)`?

Reply:
also a good call.  done